docs/metrics/ga.md

```diff
@@ -89,10 +89,14 @@ Here are the current events on the website as of this writing, grouped by their
 | Cancel during PreFeedback confirmation | button click | cancel feedback |
 | Accept Firefox permission dialog | Accept From Permission | `{experiment title}`
 | Cancel Firefox permission dialog | Cancel From Permission | `{experiment title}`
-| Send experiment app link to device | button click | Send link to device |
-| Send experiment link to email success | button | link sent to email |
-| Send experiment link to phone success | button | link sent to phone |
-| Cancel Send link to device dialog | button | cancel Send link to device dialog|
+| Send experiment app link to device | mobile send click | `{experiment title}` |
+| Click app store links on experiment page | mobile store click | `{experiment title}` `{ios || android}` |
+
+#### `SMS Modal Interactions`
+| Click app store links in modal | mobile link request | `{sms}` || `{email}` |
```

Comment:
Your table here lacks headings, so it's just a bunch of text in the rendered output.
Also, something is funny in your ``` | `{sms}` || `{email}`| ``` line, since it doesn't know if you're adding bonus columns or using an `||` OR style operator. You may need to escape w/ some ` ``` ` magic. (or just use `{sms || email }`)

Reply:
Oh whoops, nice catch